controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -195,28 +196,39 @@ public Instance require(ApplicationId id) {
         return curator.readApplications();
     }
 
-    /** Returns a snapshot of all applications */
+    /** Returns all applications of a tenant */
+    public List<Application> applicationList(TenantName tenant) {
+        return curator.readApplications(tenant);
+    }
+
+    /** Returns a snapshot of all instances */
     public List<Instance> asList() {
```

Comment:
Maybe `instanceList`?

Reply:
Yes. Probably some treatment for lock methods, too.